src/main/java/parser/Parser.java

```diff
@@ -0,0 +1,103 @@
+package parser;
```

Comment:
Should add all files to a package duke as per week 3 requirements. For eg. duke.parser